Description: remove superfluous compiler flags
 Since we build on target architecture we don't need -m32/-m64 feature
 of gcc compiler, especially since it isn't supported on all archs.
Author: Bartosz Fenski <fenio@debian.org>
Forwarded: not-needed

--- nacl-20110221.orig/okcompilers/c
+++ nacl-20110221/okcompilers/c
@@ -1,8 +1,5 @@
-gcc -m64 -O3 -fomit-frame-pointer -funroll-loops
-gcc -m64 -O -fomit-frame-pointer
-gcc -m64 -fomit-frame-pointer
-gcc -m32 -O3 -fomit-frame-pointer -funroll-loops
-gcc -m32 -O -fomit-frame-pointer
-gcc -m32 -fomit-frame-pointer
+gcc -O3 -fomit-frame-pointer -funroll-loops
+gcc -O -fomit-frame-pointer
+gcc -fomit-frame-pointer
 spu-gcc -mstdmain -march=cell -O3 -funroll-loops -fomit-frame-pointer -Drandom=rand -Dsrandom=srand
 spu-gcc -mstdmain -march=cell -O -fomit-frame-pointer -Drandom=rand -Dsrandom=srand
--- nacl-20110221.orig/okcompilers/cpp
+++ nacl-20110221/okcompilers/cpp
@@ -1,8 +1,5 @@
-g++ -m64 -O3 -fomit-frame-pointer -funroll-loops
-g++ -m64 -O -fomit-frame-pointer
-g++ -m64 -fomit-frame-pointer
-g++ -m32 -O3 -fomit-frame-pointer -funroll-loops
-g++ -m32 -O -fomit-frame-pointer
-g++ -m32 -fomit-frame-pointer
+g++ -O3 -fomit-frame-pointer -funroll-loops
+g++ -O -fomit-frame-pointer
+g++ -fomit-frame-pointer
 spu-g++ -mstdmain -march=cell -O3 -funroll-loops -fomit-frame-pointer -Drandom=rand -Dsrandom=srand
 spu-g++ -mstdmain -march=cell -O -fomit-frame-pointer -Drandom=rand -Dsrandom=srand
